client/src/modules/authorization/model/reducer.js

```diff
@@ -4,7 +4,9 @@ const currentUser = (state = null, action) => {
   switch (action.type) {
     case AuthorizationActionTypes.SET_CURRENT_USER_SUCCESS:
       return action.payload;
-    case AuthorizationActionTypes.LOGOUT_USER_SUCCESS:
+    case AuthorizationActionTypes.LOGIN_SUCCESS:
```

Comment:
Could be just moved to line 6 and then both cases will return same as now but without additional code duplication